packages/client/src/forms/certificate/fieldDefinitions/collectorSection.ts

```diff
@@ -36,7 +36,6 @@ import { identityHelperTextMapper, identityNameMapper } from './messages'
 import { Event } from '@client/utils/gateway'
 import { IDeclaration } from '@client/declarations'
 import { issueMessages } from '@client/i18n/messages/issueCertificate'
-import { labelFormatterForInformant } from '@client/views/CorrectionForm/utils'
```

Comment:
Remember to also remove this function if it's not used anymore after these changes please

Reply:
@anamulhaquemollah https://github.com/opencrvs/opencrvs-core/blob/653c045452e79c5dacb41f8e31ea14919dccf5a1/packages/client/src/views/RecordAudit/ActionDetailsModal.tsx#L156 this is the only other place where this particular function is being used and the copy for that is also similar. So could you please update the copy here as well and then we can just get rid of this function altogether

Reply:
will update accordingly